### ISSUE TITLE
Okteto:Up fails if current k8s context does not specify a namespace

### ISSUE THREAD

**Issue by pchico83** [okteto/remote-kubernetes#26]
**Describe the bug**
If my k8s context does not specify a namespace, Okteto:Up fails to load the k8s configuration.
For example, for local installations, the namespace is not specified in most cases, which means people is deploying to the *default* namespace.

**To Reproduce**
1. Switch to a k8s context without namespace
2. Okteto:Up

<img width="698" alt="Screenshot 2019-09-20 at 10 10 00" src="https://user-images.githubusercontent.com/7474696/65310733-41aa5900-db8f-11e9-86b4-c02b3efa631a.png">

**Expected behavior**
Okteto:Up should be executed in the `default` namespace

**Comment by pchico83** [comment]
```
  - context:
      cluster: minikube
      user: minikube
    name: minikube
  - context:
      cluster: okteto_pablo_okteto_net-cluster
      namespace: pchico83
      user: 777c6ade-xxxxxxxxxxxxx-02fe2e052e54
    name: okteto_pablo_okteto_net-context
```

**Comment by rberrelleza** [comment]
fix will ship in v0.1.4
